rest/blip_sync.go

```diff
@@ -932,30 +936,46 @@ func (bh *blipHandler) handleRev(rq *blip.Message) error {
 			deltaSrcBody[db.BodyAttachments] = map[string]interface{}(deltaSrcRev.Attachments)
 		}
 
-		deltaSrcMap := map[string]interface{}(deltaSrcBody)
-		err = base.Patch(&deltaSrcMap, newDoc.Body())
+		var body db.Body
```

Comment:
Can we call it `delta` instead, since that's what it represents. Having 3 bodies that represent different things floating around this bit of code is just confusing.

It isn't really a `db.Body` either (`// The body of a CouchDB document/revision as decoded from JSON`). Let's change to just `map[string]interface{}`